clients/e-shop-demo/e-shop-frontend/src/services/verify-credential.service.ts

```diff
@@ -0,0 +1,32 @@
+import { client } from '../utils/axios-client';
+
+export const readJSON = (file: File): Promise<any> => {
+	return new Promise((resolve, reject) => {
+		const fileReader = new FileReader();
+		fileReader.onload = () => resolve(JSON.parse(fileReader.result as string));
+		fileReader.onerror = () => reject;
+		fileReader.readAsText(file);
+	});
+};
+
+export const verifyCredential = async (credential: any): Promise<boolean> => {
+	try {
+		const url = `${process.env.REACT_APP_E_SHOP_BACKEND_URL}/check-credential`;
+		const response = await client.post(url, credential);
+		const isVerified = response.data.isVerified;
+		return isVerified;
+	} catch (error) {
+		console.log(error);
+		return false;
+	}
+};
+
+export const checkAge = (credential: any, ageRestriction = 18) => {
```

Comment:
The method would be easier to understand if it is named as what it is returning like `isOverAgeRestriction`